core/arch/riscv/plat-virt/conf.mk

```diff
@@ -28,7 +28,10 @@ $(call force,CFG_BOOT_SYNC_CPU,n)
 
 $(call force,CFG_RISCV_M_MODE,n)
 $(call force,CFG_RISCV_S_MODE,y)
-$(call force,CFG_RISCV_PLIC,y)
+$(call force,CFG_RISCV_PLIC,n)
+$(call force,CFG_RISCV_APLIC,n)
+$(call force,CFG_RISCV_APLIC_MSI,y)
+$(call force,CFG_RISCV_IMSIC,y)
```

Comment:
These configurations should be build time configurable.
I have proposed https://github.com/OP-TEE/optee_os/pull/7278 for `CFG_RISCV_PLIC`.
Please consider the following code:
```diff
CFG_RISCV_PLIC ?= y
+ CFG_RISCV_APLIC ?= n
+ CFG_RISCV_APLIC_MSI ?= n
+ CFG_RISCV_IMSIC ?= n
```

Thus, the developers can select different interrupt controllers when building OP-TEE OS.

Reply:
The PR is just merged today. Please rebase.